### PR TITLE
chore: update etcd version

### LINKED ISSUE #229: Why not updating etcd version?
I've seen this helm charts are using a version from mid 2020, is there a reason for that? If not, why not upgrade it? i believe it'll fix current etcd issues like #197

### PR DESCRIPTION
**Pull request by zaunist**
fix #229

### CODE REVIEW

**Comment by Gallardot** [comment]
Refer to #262. Some `etcd` configurations should be updated after upgrading the `etcd chart` version.

**Comment by zaunist** [comment]
> Refer to #262. Some `etcd` configurations should be updated after upgrading the `etcd chart` version.

Thanks, I will update configuration later.

**Comment by tao12345666333** [comment]
Please see failed jobs in CI, thanks!

File: charts/apisix/Chart.lock [hunk 1]
```diff
@@ -1,12 +1,12 @@
 dependencies:
 - name: etcd
   repository: https://charts.bitnami.com/bitnami
-  version: 6.2.6
+  version: 7.0.1
```
Comment (tao12345666333) [inline charts/apisix/Chart.lock:4]: 7.0.4 has been released, can you upgrade to the latest version?

**Review by tao12345666333 — APPROVED** [review]

**Review by Gallardot — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**